docs/errors/E0711.md

```diff
@@ -0,0 +1,22 @@
+# E0711: missing expression in placeholder within template literal
+
+Everytime you use `${}` within a template literal, which is a string defined
+with backticks, such as this one below, you should specify an expression
+inside its curly braces.
+
+```javascript
+const myString = `Here is a string defined with backticks`;
+```
+
+See an example:
+
```

Comment:
Must fix: The first code example in the file should demonstrate the error. Only your second code example demonstrates the error.
```suggestion
```